templates/vue-starter-template/app/components/account/order/Line.vue

```diff
@@ -0,0 +1,171 @@
+<script setup lang="ts">
```

Comment:
is this component named `Line`? 
For what i understand it's something like `OrdereView` or `OrderDetails`